app/models/katello/content_view_version_import_history.rb

```diff
@@ -32,11 +33,11 @@ class ContentViewVersionImportHistory < Katello::Model
     scoped_search :on => :import_type, :rename => :type, :complete_value => ContentViewVersionExportHistory::EXPORT_TYPES
 
     def self.generate_audit_comment(user:, path:, content_view_name:)
-      _("Content imported from %{path} into content view '%{name}' by %{user}") % {
+      truncate_audit_comment(_("Content imported from %{path} into content view '%{name}' by %{user}") % {
```

Comment:
I like this truncation idea. I also wonder if we can take the path out of this comment since it shows up as a property anyway?
![Audits](https://user-images.githubusercontent.com/1069779/154342962-01813408-0eeb-4b42-b644-a12c1f229d98.png)

May be we could put the content view name also as a property.

Reply:
And may be add like "Generated" : `Yes/No ` just to highlight

Reply:
Worked well in my test.

Reply:
> May be we could put the content view name also as a property.

In order to add properties to that table I think I'd have to add fields to the model. I did remove the `path` from the comment though.